### PR TITLE
Get header and user data from Doppler

### PR DESCRIPTION
**Pull request by cbernat**
### Overview
Added get from doppler correct url, and removed hardcoded data for other enviroments that are not local. 
Also added some tests for doppler legacy client.

### CODE REVIEW

**Comment by mvillaseco** [comment]
👍

File: src/services/doppler-legacy-client.ts [hunk 1]
```diff
@@ -98,13 +98,13 @@ export function mapHeaderDataJson(json: any) {
         text: json.alert.button.text,
         url: json.alert.button.url,
       },
-      message: json.alert.type,
+      message: json.alert.message,
```
Comment (andresmoschini) [inline src/services/doppler-legacy-client.ts]: Ooops, it was mine, sorry

File: src/services/doppler-legacy-client.ts [hunk 2]
```diff
@@ -98,13 +98,13 @@ export function mapHeaderDataJson(json: any) {
...
       type: json.alert.type,
     },
     nav: (json.nav && json.nav.map(mapNavMainEntry)) || [],
     user: {
       avatar: json.user.avatar,
-      email: json.email,
+      email: json.user.email,
```
Comment (andresmoschini) [inline src/services/doppler-legacy-client.ts]: What a shame!, sorry again.

File: src/services/doppler-legacy-client.ts [hunk 3]
```diff
@@ -125,21 +125,14 @@ export class HttpDopplerLegacyClient implements DopplerLegacyClient {
...
+    var response = await this.axios.get('/WebApp/GetUserData');
+    if (!response || !response.data) {
       throw new Error('Empty Doppler response');
     }
+    if (response.data.error) {
+      throw new Error(`Doppler Error: ${response.data.error}`);
+    }
 
-    // TODO: get this data from backend
-    const { alert, nav, user } = mapHeaderDataJson(headerDataJson);
```
Comment (andresmoschini) [inline src/services/doppler-legacy-client.ts]: Did you remove the dependency on `headerDataJson`?

**Comment by andresmoschini** [comment]
Great work! thanks.

File: src/services/doppler-legacy-client.ts (hunk 2 above)
Comment (cbernat) [reply src/services/doppler-legacy-client.ts]: It happens sometimes :) Don't worry much.

File: src/services/doppler-legacy-client.ts (hunk 3 above)
Comment (cbernat) [reply src/services/doppler-legacy-client.ts]: No, removing now...

**Pull request merged**